packages/block-editor/src/components/block-draggable/style.scss

```diff
@@ -16,6 +16,9 @@
 	padding: 0 ( $grid-unit-15 + $border-width );
 	user-select: none;
 	width: max-content;
+	transition:
+		background-color 0.5s linear,
```

Comment:
I suggest something much quicker, but with a tiny delay (to reduce the flashes): 

`background-color 0.1s linear 0.1s, opacity 0.1s linear 0.1s;`

Reply:
While we're here, a tiny tweak to the chip's shadow makes it feel nice :) 

`box-shadow: 0 4px 8px rgba($black, 0.2);`

Reply:
Thanks, added!